src/handlers/class-check-changes-handler.php

```diff
@@ -108,12 +108,15 @@ public function check_changes_action_handler() {
 						<div class="diff">
 						<?php
 						$fields = [
-							\__( 'Title', 'default' )   => 'post_title',
-							\__( 'Content', 'default' ) => 'post_content',
-							\__( 'Excerpt', 'default' ) => 'post_excerpt',
+							'post_title'   => \__( 'Title', 'default' ),
+							'post_content' => \__( 'Content', 'default' ),
+							'post_excerpt' => \__( 'Excerpt', 'default' ),
 						];
 
-						foreach ( $fields as $name => $field ) {
+						$post_array = \get_post( $post, \ARRAY_A );
+						$fields     = \apply_filters( '_wp_post_revision_fields', $fields, $post_array ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
```

Comment:
Please move the ignore to the line above, and add a reason by adding `-- Reason: ...` at the end